### ISSUE TITLE
GTA Vice City: some textures not scaling up properly

### ISSUE THREAD

**Issue by OrganicOstrich** [xemu-project/xemu#1658]
### Title

https://xemu.app/titles/5454000f/#Grand-Theft-Auto-Vice-City

### Bug Description

![20240323_172805](https://github.com/xemu-project/xemu/assets/163926395/063a88d7-4223-4dde-bdf3-d15d653fec9a)
![20240323_172821](https://github.com/xemu-project/xemu/assets/163926395/33deb182-e4af-44c4-b91c-02bbbca6630c)

### Expected Behavior

https://gta.fandom.com/fr/wiki/Enforcer?file=Enforcer_GTAVC_rear.jpg

### xemu Version

0.7.122

### System Information

CPU:          Intel(R) Core(TM) i5-9300H CPU @ 2.40GHz
OS Platform:  Windows
OS Version:    23H2
Manufacturer: NVIDIA Corporation
GPU Model:    NVIDIA GeForce GTX 1650/PCIe/SSE2
Driver:       4.0.0 NVIDIA 552.22
Shader:       4.00 NVIDIA via Cg compiler

**Comment by RinMaru** [comment]
how about capturing on actual hw and not on some site that has any version of the game? @mborgerson dont think this is an actual issue unless they can provide blatant proof this isn't just how it look on real hw because that looks fine here

**Comment by mborgerson** [comment]
Agreed, this issue is not very clear. Can you give more details about what you expect to see? If you mean the pixelated Vice City Police Department decal, I think it is expected. The stray lines on the VCPD decal may be an artifact, like Halo menu.

**Comment by OrganicOstrich** [comment]
> how about capturing on actual hw and not on some site that has any version of the game? [@mborgerson](https://github.com/mborgerson) dont think this is an actual issue unless they can provide blatant proof this isn't just how it look on real hw because that looks fine here

I don't currently have a working copy of the game. I have found footage on YouTube of someone playing on Xbox 360 via emulation, and whilst not technically real hardware, the glitch is still present:

https://www.youtube.com/watch?v=YR0LxAo9d3I  7:24

This is not "fine": it is a glitch where the crest in the VCPD logo is enlarged and  engulfs the rest of the badge. This is not present in any other versions of the game but, since the glitch is native to the hardware, it is something that is already broken and is not the responsibility of Xemu. Usually when something is more blatant, I can read old reviews and forums, and mention it in my compatibility reports. Since I found no mention of it, I just assume it was an emulation glitch.

**Comment by OrganicOstrich** [comment]
> Agreed, this issue is not very clear. Can you give more details about what you expect to see? If you mean the pixelated Vice City Police Department decal, I think it is expected. The stray lines on the VCPD decal may be an artefact, like Halo menu.

I wasn't saying about it being pixelated, I was saying that the police crest is not meant to look like that as the badge has been enlarged massively,  to the point that it is covering the rest and clipping into the letter.

It is a bug, but as mentioned above, is native to Xbox hardware

**Comment by OrganicOstrich** [comment]
https://www.youtube.com/watch?v=jaaDMDB0Mcw 7:24

Not actual hardware but still present so clearly this glitch is native
